### PR TITLE
Text is placed in the centre of screen . Issue #390

### PR DESCRIPTION
**Pull request by parasg97**
Converted the layout from linear to more efficient constraint layout

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -116,6 +116,7 @@ dependencies {
   // Mockito
   testImplementation "org.mockito:mockito-core:2.7.22"
   androidTestImplementation "org.mockito:mockito-android:2.7.22"
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```
Comment (mhutti1) [inline app/build.gradle]: Incorrect indentation (also is this needed? We might have elsewhere)

File: app/src/main/res/layout/activity_library.xml [hunk 2]
```diff
@@ -1,48 +1,54 @@
...
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?attr/listBackground">
+    android:orientation="vertical"
+    tools:context=".zim_manager.library_view.LibraryFragment"
+    android:id="@+id/relativeLayout">
```
Comment (srv-twry) [inline app/src/main/res/layout/activity_library.xml]: Rename the id.

File: app/src/main/res/layout/activity_library.xml [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
-
-  <RelativeLayout
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?attr/listBackground">
+    android:orientation="vertical"
```
Comment (srv-twry) [inline app/src/main/res/layout/activity_library.xml]: Orientation doesn't mean anything in ConstraintLayout.

File: app/src/main/res/layout/activity_library.xml [hunk 4]
```diff
@@ -1,48 +1,54 @@
...
+                android:id="@+id/library_list"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:paddingBottom="60dp"
+                android:background="?attr/listBackground"
+                android:divider="@null"/>
+
+        </android.support.v4.widget.SwipeRefreshLayout>
+
+
```
Comment (srv-twry) [inline app/src/main/res/layout/activity_library.xml]: Avoid unnecessary spaces.

**Comment by srv-twry** [comment]
Also please link the PR with the issue so that it is automatically closed once the PR is merged.

**Comment by srv-twry** [comment]
Closing due to inactivity. Please reopen it once ready.

**Pull request closed without merging**